tempto-core/src/main/java/io/prestosql/tempto/internal/convention/ConventionBasedTestProxyGenerator.java

```diff
@@ -82,7 +82,10 @@ public ConventionBasedTest generateProxy(ConventionBasedTest conventionBasedTest
 
     private String generatedClassName(ConventionBasedTest conventionBasedTest)
     {
-        List<String> testNameParts = Splitter.on('.').splitToList(conventionBasedTest.getTestName());
+        List<String> testNameParts = Splitter.on('.').omitEmptyStrings().splitToList(conventionBasedTest.getTestName());
+        if (testNameParts.size() == 1) {
+            return testPackage;
```

Comment:
Can this lead to name collisions?

Reply:
To be honest, I don't really understand the point of this code. Why do we only use the last part of the name? But this fixes the crash and doesn't seem to cause other issues.